README.md

```diff
@@ -39,6 +39,23 @@ port that you want the gateway to listen on (default is 8080).
 
 Start the gateway simply by running the executable `oasis-core-rosetta-gateway`.
 
+## Offline mode
+
+The gateway supports an "offline" mode, which enables only a subset of the
+Construction API and nothing else, but doesn't require a connection to an
+Oasis node.
+
+To enable it, set the environment variable `OASIS_ROSETTA_GATEWAY_OFFLINE_MODE`
+to a non-empty value.  You must also set the environment variable
+`OASIS_ROSETTA_GATEWAY_OFFLINE_MODE_CHAIN_ID` to the chain ID of the network
```

Comment:
Maybe this should be called something else than "chain ID" as in the documentation we call it the ["genesis document hash"](https://docs.oasis.dev/oasis-core/high-level-components/index/genesis).

Reply:
I've renamed it to "genesis document hash" in the README, so it matches the current docs.